### ISSUE TITLE
First LED of Matrix is flashing/flickering green 

### ISSUE THREAD

**Issue by Puh73** [AlexGyver/GyverLamp#26]
I have the problem, that the first LED of my 16x16 LED Matrix is flashing/flickering green. I already have googled this problem and found out that I am not the only one with this problem. In Internet I found many people who have problems with NodeMCU, ESP8266, Wemos D1, and other boards and WS2812 LED-Strips with flashing/flickering green first LED-Pixel and no one have solved this issue. I saw that you have added an resistor and diode on LED-Strip of GyverLamp2 but with no description why and I think the diode don’t is in the right direction. So I hope you can help or describe why you have done this and only on GyverLamp2 and not change the scheme of GyverLamp. Am I the only one with this flashing first LED-Pixel problem? Thank you in advance. 

Patrick

**Comment by camohob** [comment]
+1, i have the same problem. @Puh73 did you solve this problem?

**Comment by Puh73** [comment]
> +1, i have the same problem. @Puh73 did you solve this problem?

Yes, the problem was caused by the ESP8266 Driver. You have to switch to an older driver in Arduino IDE than everything works fine.

**Comment by gigby** [comment]
@Puh73 I have the same issue. I installed 3.0.2v of the ESP8266 driver for my mac m1 (suggested version 2.5.2 got error during compilation because coulndn't find python (and a magic with symlinks didn't help me)).
Which version did you use to fix this issue?

**Comment by gigby** [comment]
The issue fixed via installing esp8266 v2.6.0. Now everything works stable.
First of all, I was trying to install fork from Gunner (86 in 1).
When I followed instructions from archive (installed esp v2.5.2 etc) I got issue with broken symlink and loosing wi-fi signal when you're changing effect from expanded list (esp_mode=1). Symlink to python was fixed but it was impossible to fix wi-fi issue.
PS: I have mac os Ventura 13.0.1, python v3.11.0 and latest version of Gunner47 firmware. Probably current project (original) doesn't have wi-fi issue at all.
